orderbook/src/api/order_validation.rs

```diff
@@ -0,0 +1,734 @@
+use crate::{account_balances::BalanceFetching, fee::MinFeeCalculating};
```

Comment:
I like that it's its own component now :smile:

Reply:
Me too!